api/launchpad/v1/launchpad.proto

```diff
@@ -0,0 +1,161 @@
+syntax = "proto3";
+
+package launchpad.v1;
+option go_package = "./launchpadpb";
+
+service LaunchpadService {
+  rpc UploadMetadatas(UploadMetadatasRequest) returns (UploadMetadatasResponse);
+  rpc CalculateCollectionMerkleRoot(CalculateCollectionMerkleRootRequest) returns (CalculateCollectionMerkleRootResponse);
+  rpc TokenMetadata(TokenMetadataRequest) returns (TokenMetadataResponse);
+  rpc LaunchpadProjectsByCreator(LaunchpadProjectsByCreatorRequest) returns (LaunchpadProjectsByCreatorResponse);
+  rpc LaunchpadProjects(LaunchpadProjectsRequest) returns (LaunchpadProjectsResponse);
+  rpc LaunchpadProjectById(LaunchpadProjectByIdRequest) returns (LaunchpadProjectByIdResponse);
+  rpc LaunchpadProjectsCounts(LaunchpadProjectsCountsRequest) returns (LaunchpadProjectsCountsResponse);
+  rpc ProposeApproveProject(ProposeApproveProjectRequest) returns (ProposeApproveProjectResponse);
+}
+
+enum Sort {
+  SORT_UNSPECIFIED = 0;

```

Comment:
having this is idiomatic protobuf. To answer, since we don't specify, the behavior is implementation-specific, but yeah in most case it will select a default or error out